src/body.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
change the name of the file - `reset.css`, instead `body.css`
This file will reset the properties and nothing more